presto-spi/src/main/java/com/facebook/presto/spi/page/SerializedPage.java

```diff
@@ -84,13 +85,31 @@ public Slice getSlice()
     }
 
     @Override
-    public String toString()
+    public boolean equals(Object o)
```

Comment:
Why to replace `toString` with the `equals` and `hashCode` pair?

Reply:
- `toString()` is not used, removed to avoid using guava
- `equals()` and `hashCode()` is actually missing in test and should be added